Orange/widgets/data/owmergedata.py

```diff
@@ -369,7 +369,11 @@ def _join_table_by_indices(self, reduced_extra, indices):
         string_cols = [i for i, var in enumerate(domain.metas) if var.is_string]
         metas = self._join_array_by_indices(
             self.data.metas, reduced_extra.metas, indices, string_cols)
-        return Orange.data.Table.from_numpy(domain, X, Y, metas)
+        table = Orange.data.Table.from_numpy(domain, X, Y, metas)
+        table.name = getattr(self.data, 'name', '')
+        table.attributes = getattr(self.data, 'attributes', '')
```

Comment:
Default for `attributes` should be `{}`.

(When fixed, don't make a new commit but amend this one, then `push -f`)